.github/PULL_REQUEST_TEMPLATE/integration.md

```diff
@@ -8,30 +8,30 @@ This PR adds support for [`<integration>`](<!--link to relevant integration docs
 - Corp docs PR: <!-- add link here -->
 
 ## Checklist
+<!-- Delete any entries that are not applicable to the integration -->
+
 - [ ] Usage and configuration documentation added in `__init__.py`.
-- [ ] [Corp docs](https://github.com/Datadog/documentation) are updated.
-- [ ] Distributed tracing propagation is implemented (if applicable).
+- [ ] [Corp docs](https://github.com/Datadog/documentation) PR to add new integration to documentation.
 - [ ] Span metadata
-  - [ ] Span type
+  - [ ] Service (use [`int_service`](https://github.com/DataDog/dd-trace-py/blob/90d1d5981c72ea312c21ac04e5be47521d0f0f2e/ddtrace/contrib/trace_utils.py#L55) or [`ext_service`](https://github.com/DataDog/dd-trace-py/blob/90d1d5981c72ea312c21ac04e5be47521d0f0f2e/ddtrace/contrib/trace_utils.py#L87).
+  - [ ] Span type should be one of [these](https://github.com/DataDog/dd-trace-py/blob/90d1d5981c72ea312c21ac04e5be47521d0f0f2e/ddtrace/ext/__init__.py#L7).
   - [ ] Resource
 - [ ] Global configuration
-  - [ ] Inherits `DD_SERVICE` (if applicable).
```

Comment:
This is now managed by the trace util service helpers mentioned above for service

Reply:
💯